pulp-glue/pulp_glue/ansible/context.py

```diff
@@ -17,6 +16,8 @@
 
 
 class PulpAnsibleCollectionVersionContext(PulpContentContext):
+    PLUGIN = "ansible"
+    RESOURCE_TYPE = "collection_version"
```

Comment:
In the end this is a user facing string (still an identifier, so no, not translatable). Do we want to use `-` instead of `_` here?

Reply:
I don't have a strong opinion either way (amazing, I know...) Obviously, if we decide '-', then we do it for all the RESOURCE_TYPEs - but I am OK with this kind-of identifier looking more like an identifier, than a human concept.